pages/buildkite_compute/linux_instances.md

```diff
@@ -0,0 +1,41 @@
+## Linux compute instances
+
+Linux instances are offered with two architectures.
+
+- ARM
+- AMD64 (x64_86)
+
+To accommodate different workloads, instances are capable of running up to 8 hours. If you require longer running agents please contact support.
+
+### Size
+
+We offer a selection of instance sizes, allowing you to tailor your compute resources to the demands of your jobs. Below is a breakdown of the available sizes.
+
+<table>
+    <thead>
+        <tr><th>Size</th><th>vCPU</th><th>RAM</th></tr>
+    </thead>
+    <tbody>
+        <tr><td>Small</td><td>2</td><td>4 GB</td></tr>
+        <tr><td>Medium</td><td>4</td><td>8 GB</td></tr>
+        <tr><td>Large</td><td>8</td><td>32 GB</td></tr>
+    </tbody>
+</table>
+
+### Image configuration
+To configure your Linux instance you can use the [Docker Compose](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) plugin.
```

Comment:
For improved source code readability, please ensure all headings are surrounded by a single empty line.

```suggestion
### Image configuration

To configure your Linux instance you can use the [Docker Compose](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) plugin.
```